mmengine/hooks/runtime_info_hook.py

```diff
@@ -23,7 +23,7 @@ def _is_scalar(value: Any) -> bool:
     """
     if isinstance(value, np.ndarray):
         return value.size == 1
-    elif isinstance(value, (int, float)):
+    elif isinstance(value, (int, float, np.floating, np.integer)):
```

Comment:
Can we replace `np.floating, np.integer` with `np.number`?

Reply:
@zhouzaida  Thank you for your comment. Yes, I missed it. 
The np.number would be better.

For clarification, a difference is that np.number includes complex numbers while the `np.floating, np.integer` does not.
https://numpy.org/doc/stable/reference/arrays.scalars.html